<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>74</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSpinBox" name="tailSec">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="pBar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="tailSecText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>cut tail sec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileLine">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>video_file</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
